link_telemetry.py

```diff
@@ -705,9 +718,15 @@ def main():
                     # read in bytes from COM port
                     chunk = ser.read(CHUNK_SIZE)
                     chunk = chunk.hex()
+
+                    if args.rawest:
```

Comment:
I think it would be good to make it very clear that the `args.rawest` prints can have incomplete messages. Can you please add their usage (as well as this warning) in the README